src/util/metamask.js

```diff
@@ -1,6 +1,33 @@
+import ENS from "ethjs-ens";
+
+
 export const assignEthereum = () => {
     if (typeof window.ethereum !== 'undefined') {
         return window.ethereum
     }
     return false
-}
\ No newline at end of file
+}
+
+const ens = new ENS({ provider: window.web3.currentProvider, network: 1 });
+
+export const convertENS = async (value) => {
```

Comment:
Why is this all in `metamask.js`? This has nothing to do with metamask. Maybe move it to a new util file for ens?

Reply:
it was in its file but I followed your base master branch, will make it into its own file